### PR TITLE
Update finish scripts for s6 v3

### PR DESCRIPTION
**Pull request by mdegat01**
Update finish scripts for https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images

For most addons did not bump the version, just ensuring the next PR to go in doesn't have a surprise issue. Did bump the version for ZwaveJS and Deconz since they have all had recent updates so want to make new images without it.

### CODE REVIEW

File: deconz/config.yaml
Comment (pvizeli) [inline deconz/config.yaml:2]: please add codenotary

File: zwave_js/config.yaml
Comment (pvizeli) [inline zwave_js/config.yaml:2]: codenotary

**Review by pvizeli — APPROVED** [review]

File: ada/rootfs/etc/services.d/ada/finish
Comment (frenck) [inline ada/rootfs/etc/services.d/ada/finish:9]: Otherwise it will continue the the restarting message 

```suggestion
  exec /run/s6/basedir/bin/halt
```

File: almond/rootfs/etc/services.d/almond/finish
Comment (frenck) [inline almond/rootfs/etc/services.d/almond/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: almond/rootfs/etc/services.d/nginx/finish
Comment (frenck) [inline almond/rootfs/etc/services.d/nginx/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: configurator/rootfs/etc/services.d/configurator/finish
Comment (frenck) [inline configurator/rootfs/etc/services.d/configurator/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: deconz/rootfs/etc/services.d/deconz/finish
Comment (frenck) [inline deconz/rootfs/etc/services.d/deconz/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: deconz/rootfs/etc/services.d/nginx/finish
Comment (frenck) [inline deconz/rootfs/etc/services.d/nginx/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: google_assistant/rootfs/etc/services.d/google-assistant/finish
Comment (frenck) [inline google_assistant/rootfs/etc/services.d/google-assistant/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: homematic/rootfs/etc/services.d/hmserver/finish
Comment (frenck) [inline homematic/rootfs/etc/services.d/hmserver/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: homematic/rootfs/etc/services.d/nginx/finish
Comment (frenck) [inline homematic/rootfs/etc/services.d/nginx/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: homematic/rootfs/etc/services.d/regahss/finish
Comment (frenck) [inline homematic/rootfs/etc/services.d/regahss/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: homematic/rootfs/etc/services.d/rfd/finish
Comment (frenck) [inline homematic/rootfs/etc/services.d/rfd/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: homematic/rootfs/etc/services.d/wired/finish
Comment (frenck) [inline homematic/rootfs/etc/services.d/wired/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: mariadb/rootfs/etc/services.d/mariadb/finish
Comment (frenck) [inline mariadb/rootfs/etc/services.d/mariadb/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: mosquitto/rootfs/etc/services.d/mosquitto/finish
Comment (frenck) [inline mosquitto/rootfs/etc/services.d/mosquitto/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: mosquitto/rootfs/etc/services.d/nginx/finish
Comment (frenck) [inline mosquitto/rootfs/etc/services.d/nginx/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: ssh/rootfs/etc/services.d/sshd/finish
Comment (frenck) [inline ssh/rootfs/etc/services.d/sshd/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: ssh/rootfs/etc/services.d/ttyd/finish
Comment (frenck) [inline ssh/rootfs/etc/services.d/ttyd/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: vlc/rootfs/etc/services.d/nginx/finish
Comment (frenck) [inline vlc/rootfs/etc/services.d/nginx/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: vlc/rootfs/etc/services.d/vlc/finish
Comment (frenck) [inline vlc/rootfs/etc/services.d/vlc/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: zwave/rootfs/etc/services.d/mosquitto/finish
Comment (frenck) [inline zwave/rootfs/etc/services.d/mosquitto/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: zwave/rootfs/etc/services.d/ozwadmin/finish
Comment (frenck) [inline zwave/rootfs/etc/services.d/ozwadmin/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: zwave/rootfs/etc/services.d/websockify/finish
Comment (frenck) [inline zwave/rootfs/etc/services.d/websockify/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: zwave/rootfs/etc/services.d/zwave/finish
Comment (frenck) [inline zwave/rootfs/etc/services.d/zwave/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: zwave_js/rootfs/etc/services.d/zwave_js/finish
Comment (frenck) [inline zwave_js/rootfs/etc/services.d/zwave_js/finish:9]:
```suggestion
  exec /run/s6/basedir/bin/halt
```

File: ada/rootfs/etc/services.d/ada/finish
Comment (mdegat01) [reply ada/rootfs/etc/services.d/ada/finish:9]: Oh I might have to fix others then. I just copied what we did in [the example addon](https://github.com/home-assistant/addons-example/blob/main/example/rootfs/etc/services.d/example/finish). We also maybe should update the [blog post](https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images#minimum) again @pvizeli .

**Comment by mdegat01** [comment]
@pvizeli After this one is merged I'll do another PR pass to add codenotary to all the ones missing it.

**Review by frenck — APPROVED** [review]
Thanks, @mdegat01 👍 

I think this looks good and can be merged (if we follow up with the codenotary change as suggested by Pascal in follow-up PR).

**Pull request merged**
